src/wolfgame.py

```diff
@@ -3879,6 +4011,9 @@ def check_exchange(cli, actor, nick):
             if nick_role == "shaman":
                 pm(cli, actor, "You have a \u0002{0}\u0002 totem.".format(nick_totem))
             var.TOTEMS[actor] = nick_totem
+        elif nick_role == "mystic":
+            numevil = len(var.list_players(var.WOLFTEAM_ROLES))
+            pm(cli, actor, "There are \u0002{0}\u0002 evil villager{1} still alive.".format(numevil, "s" if numevil != 1 else ""))
```

Comment:
Erm, seems like that will say "There are 1 evil villager still alive." Should be "is".